src/components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { Css, IconButton, useTestIds } from "src/index";
+import { ReactNode, useState } from "react";
+
+export interface AccordionProps {
+  title: string;
+  children: ReactNode;
+  disabled?: boolean;
+  defaultExpanded?: boolean;
+  bottomSection?: boolean;
+}
+
+export function Accordion(props: AccordionProps) {
+  const { title, children, disabled = false, defaultExpanded = false, bottomSection = false} = props;
+  const [expanded, setExpanded] = useState(defaultExpanded);
+  const [focused, setFocused] = useState(false);
```

Comment:
**suggestion:** I don't believe we want the focus state to always show when focused. It should be a keyboard sort of focus (like showing an outline when you hit tab vs showing when you click). For that, we use a helper called `useFocusRing`. You should be able to find example uses of it in the application. It is pretty straight forward, but let me know if you want any more details.